CMakeLists.txt

```diff
@@ -67,9 +67,11 @@ IF (BUILD_FUZZER)
     ENDIF()
     ADD_EXECUTABLE(quicly-fuzzer-packet fuzz/packet.cc ${PICOTLS_OPENSSL_FILES})
     SET(LIB_FUZZER "${CMAKE_CURRENT_BINARY_DIR}/libFuzzer.a")
-    SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fno-omit-frame-pointer -fsanitize=address,undefined -fsanitize-coverage=edge,indirect-calls,8bit-counters")
-    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fno-omit-frame-pointer -fsanitize=address,undefined -fsanitize-coverage=edge,indirect-calls,8bit-counters")
-    SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_C_FLAGS}")
+    SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fno-omit-frame-pointer -fsanitize=address,undefined")
```

Comment:
I see `-fsanitize-coverage=edge,indirect-calls,8bit-counter` being removed from this line and the one below. Is it necessary?

Reply:
The 8bit-counter option needs to have a function implemented with the signature "void __sanitizer_cov_8bit_counters_init(char *start, char *end)", but it doesn't look like quicly has that, so it made sense to get rid of it.

Reply:
it makes sense to keep "edge" option, so I'll put that back in.

Reply:
Ah! I see that fsanitize-coverage=edge requires the additional trace-pc option (edge by itself used as an option is deprecated, so we're forced to use trace-pc as well). However, when we also include trace-pc as an option, we then need to add the definition of void __sanitizer_cov_trace_pc(void) to the relevant places. I've added that in now.

Reply:
@deweerdt Do you have any suggestions? I ask because it seems to that the use of these compile options originate from #1.

Reply:
The new version of `oss-fuzz` doesn't require those flags.